cs/tools/TunnelsSDK.Generator/RustContractWriter.cs

```diff
@@ -64,6 +80,44 @@ public override void WriteContract(ITypeSymbol type, ICollection<ITypeSymbol> al
         File.WriteAllText(filePath, s.ToString());
     }
 
+    public override void WriteCompleted()
+    {
+        this.WriteModRs();
+    }
+
+    private void WriteModRs()
```

Comment:
Great, I had been thinking it would be nice to generate this also, thanks for adding it.